### PR TITLE
Adding sq translations files

### PR DESCRIPTION
**Pull request by ujdhesa**
This PR/Translation is about #ISSUE_NUMBER.

**files translated**:
  - [ ] `def/--LOCALE-CODE--/about.cson`
  - [ ] `def/--LOCALE-CODE--/context.cson`
  - [ ] `def/--LOCALE-CODE--/menu_darwin.cson`
  - [ ] `def/--LOCALE-CODE--/menu_linux.cson`
  - [ ] `def/--LOCALE-CODE--/menu_win32.cson`
  - [ ] `def/--LOCALE-CODE--/settings.cson`
  - [ ] `def/--LOCALE-CODE--/welcome.cson`

**validation**
  - [ ] I've validated my translation locally by running

`npm run validation -- --locale MY_LOCALE`

### CODE REVIEW

**Comment by liuderchi** [comment]
@ujdhesa thanks for your contribution to Albanian

**Comment by ujdhesa** [comment]
> @ujdhesa thanks for your contribution to Albanian for #122
> 
> How do you type Albanian in Albanian?
> Is it `gjuha shqipe` or `shqipe` ?

Obviously you’re asking how to represent it in a menu. If the screen is something like:

Language:
    Albanian
    Armenian
   ..............
   English
   ..............

it should be **shqipe**.

If the word language is shown nowhere, i.e. we're talking about a plain list of locales, then **shqip** or **Shqip** (according the case to the typography of that list) is the right one.

**Comment by ujdhesa** [comment]
menu_linux.cson added.

File: def/sq/menu_win32.cson [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+        value: "&Ribëje"
+      "Cu&t":
+        value: "&Prije"
+      "&Copy":
+        value: "&Kopjoje"
+      "Copy Pat&h":
+        value: "Kopjoji Shte&gun"
+      "&Paste":
+        value: "&Ngjite"
+      "Select &All":
```
Comment (liuderchi) [inline def/sq/menu_win32.cson]: missing `"Paste Without Reformatting"` above (see [template/menu_win32.cson](https://github.com/liuderchi/atom-i18n/blob/master/def/template/menu_win32.cson#L65))

**Comment by liuderchi** [comment]
Todo for myself:
- [ ] fix CI error (https://github.com/liuderchi/atom-i18n/commit/ced8de19fb9f19d51292fc4fdac5395b73bc1990)
- [ ] update Codeowner
- [ ] update contributor in README
- [ ] update CHANGELOG

**Comment by ujdhesa** [comment]
https://github.com/ujdhesa/atom-i18n/commit/312aacdf30337d7fcd09cfac703585552b727808 adds the missing entity.

**Pull request merged**